tests/llm/utils/mock_utils.py

```
@@ -42,6 +42,14 @@ class Message(BaseModel):
 T = TypeVar("T")
 
 
+ALLOWED_EVAL_TAGS = Literal[
```

Flake8 (7.2.0)

[error] redefinition of unused 'ALLOWED_EVAL_TAGS' from line 15

(F811)